backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5757190_sys_gh23706_keep_single_Purchase_Candidate_Price_UOM_ID.sql

```diff
@@ -0,0 +1,47 @@
+UPDATE ad_column
+SET columnname = 'Price_UOM_back_up_ID', Updated=TO_TIMESTAMP('2025-06-10 15:39:15', 'YYYY-MM-DD HH24:MI:SS'), UpdatedBy=100
+WHERE ad_column_id = 582988
+;
+
+-- Column: C_PurchaseCandidate.Price_UOM_ID
+-- Column: C_PurchaseCandidate.Price_UOM_ID
+-- 2025-01-09T17:58:13.095Z
+INSERT INTO AD_Column (AD_Client_ID,AD_Column_ID,AD_Element_ID,AD_Org_ID,AD_Reference_ID,AD_Reference_Value_ID,AD_Table_ID,AD_Val_Rule_ID,CloningStrategy,ColumnName,Created,CreatedBy,DDL_NoForeignKey,EntityType,FacetFilterSeqNo,FieldLength,IsActive,IsAdvancedText,IsAllowLogging,IsAlwaysUpdateable,IsAutoApplyValidationRule,IsAutocomplete,IsCalculated,IsDimension,IsDLMPartitionBoundary,IsEncrypted,IsExcludeFromZoomTargets,IsFacetFilter,IsForceIncludeInGeneratedModel,IsGenericZoomKeyColumn,IsGenericZoomOrigin,IsIdentifier,IsKey,IsLazyLoading,IsMandatory,IsParent,IsRestAPICustomColumn,IsSelectionColumn,IsShowFilterIncrementButtons,IsShowFilterInline,IsStaleable,IsSyncDatabase,IsTranslated,IsUpdateable,IsUseDocSequence,MaxFacetsToFetch,Name,SelectionColumnSeqNo,SeqNo,Updated,UpdatedBy,Version) VALUES (0,589574,542464,0,18,114,540861,540472,'XX','Price_UOM_ID',TO_TIMESTAMP('2025-01-09 17:58:12.811000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'N','de.metas.purchasecandidate',0,10,'Y','N','Y','N','Y','N','N','N','N','N','Y','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','N','Y','N',0,'Preiseinheit',0,0,TO_TIMESTAMP('2025-01-09 17:58:12.811000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,0)
+;
```

Comment:
This will fail on systems, where `5742770_sys_gh19759_C_PurchaseCandidate_Price_UOM_ID` was executed